expyfun/_experiment_controller.py

```diff
@@ -948,7 +955,7 @@ def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
         ExperimentController.wait_for_presses
         """
         return self._response_handler.get_presses(live_keys, timestamp,
-                                                  relative_to)
+                                                  relative_to, type)
```

Comment:
Don't shadow builtins (don't use name `type`, perhaps `kind` instead)

Also, I don't see `type` value- or type-checked anywhere, so currently a user passing a typo `type='pressees` will instead get some ambiguous `KeyError` instead of a clear message about what values are accepted